driver/gl/window.go

```diff
@@ -399,9 +417,9 @@ func (w *window) keyPressed(viewport *glfw.Window, key glfw.Key, scancode int, a
 		return
 	}
 
-	if key <= glfw.KeyWorld1 { // filter printable characters handled in charModInput
```

Comment:
I think that taking this out makes duplicate events - but I don't recall exactly.

Reply:
Refactored in 949f61f.
It is used to hande not printable character into the OnKeyDown.
Just as note: wondering if in the future next releases we could handle separately printable chars and no ones in dedicated functions aiming to shrink the entry.OnKeyDown logic.